source/xml/reading.md

```diff
@@ -381,11 +381,11 @@ their respective XML elements:
             // Borrowing a parser from the KeyValue class.
```

Comment:
Attributes do not have their namespaces prefixed _unless_ the attribute namespace is differerent from the element namespace.